### ISSUE TITLE
fs.copyfile kwargs unimplemented

### ISSUE THREAD

**Issue by apprehensions** [muon-build/muon#100]
```
import('fs').copyfile(f, f, install: true, install_dir: get_option('includedir'))
```
will only copy to itself, and not install as written.

Workaround is to construct the filepath by hand.

This is used in GLib.

**Comment by apprehensions** [comment]
A possible implementation is to construct the filemath in `copyfile.meson`, if that's necessary, i'm okay with making a futile attempt of a PR.

**Comment by apprehensions** [comment]
Implemented in 5747da334da50236a44ebe7182a1b5342f8d05d7
